js/htmldiff.js

```diff
@@ -732,32 +780,33 @@
      *
      * @param {sting} tag The tag name of the wrapper tags.
      * @param {Array.<string>} content The list of tokens to wrap.
```

Comment:
Looks like you're missing a @param for the opIndex. Also, I'm confused about the purpose of the data attribute with the op index. How is that used?